docs/eip-draft_bls_remote_signer_api.md

```diff
@@ -0,0 +1,418 @@
+---
+eip: <to be assigned>
+title: BLS Remote Signer HTTP API Standard
+author: Herman Junge <herman@sigmaprime.io>
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-09-30
+---
+
+## Simple Summary
+This EIP defines a HTTP API standard for a BLS remote signer, consumed by validator clients to sign block proposals and attestations in the context of Ethereum 2.0 (eth2).
+
+## Abstract
+A [validator](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/validator.md) client contributes to the consensus of the Eth2 blockchain by signing proposals and attestations of blocks, using a BLS private key which must be available to this client at all times.
+
+The BLS remote signer API is designed to be consumed by validator clients, looking for a more secure avenue to store their BLS12-381 private key(s), enabling them to run in more permissive and scalable environments.
+
+## Motivation
+Eth2 utilizes [BLS12-381](https://github.com/cfrg/draft-irtf-cfrg-bls-signature/) signatures.
+
+Consensus on the eth2 Blockchain is achieved via the proposal and attestation of blocks from validator clients, using a BLS private key (_signing_ key) which must be available each time a message is signed: that is, at least once every epoch (6.4 minutes), during a small window of time within this epoch (a _slot_, i.e. 12 seconds), as each validator is expected to attest exactly once per epoch.
+
+The [eth2 specification](https://github.com/ethereum/eth2.0-specs) does not explicitly provide a directive on where this BLS private key must/should be stored, leaving this implementation detail to the client teams, who assume that this cryptographic secret is stored on the same host as the validator client.
+
+This assumption is sufficient in the use case where the validator client is running in a physically secure network (i.e. nobody, but the operator, has a chance to log-in into the machine hosting the validator client), as such configuration would only allow _outbound_ calls from the validator client. In this situation, only a physical security breach can allow an attacker to either have arbitrary access to the storage or to the memory of the device.
```

Comment:
>only a physical security breach

I guess Remote Code Execution (RCE) vulnerability can also allow unauthorized access in this context. Might be worth mentioning too.

Reply:
Improved to 

> In this situation, only a physical security breach, or a Remote Code Execution (RCE) vulnerability can allow an attacker to either have arbitrary access to the storage or to the memory of the device.